src/components/Proposal/Create/PluginForms/ResetFormButton.scss

```diff
@@ -0,0 +1,24 @@
+.resetFormButton {
+  height: 30px;
+  border: none;
+  display: inline-block;
+  margin: 0 5px;
+  font-size: 13px;
+  padding: 0 25px 2px 25px;
+  font-family: $body-font;
+  font-weight: bold;
+  transition: all 0.25s ease;
+  background-color: $green-button-color;
```

Comment:
I think a red color for a 'clear' button makes more sense, what do you think?
Or maybe transparent fill but with a red border.

Reply:
Why red?

Reply:
Usually red color refers to delete/clear/reset while green color refers to add/execute.

Reply:
Done:

![image](https://user-images.githubusercontent.com/1821666/89450783-1d188a80-d729-11ea-9d7d-bbc75e0ca0ff.png)